### DISCUSSION TITLE
Api V3 Response Is Very Slow

### DISCUSSION THREAD

**Discussion by Esatyilmaz0** [etsy/open-api#829] [Q&A]
Hello, I am developing etsy api v3 application with python. But each request takes an average of 6-7 seconds, and this makes users have a bad experience. What can I do to Fix this? I added connection close and user agent header, still the result did not change.

**Comment by PMStanley** [comment]
This might be related to your distance to Etsy's servers or something to do with your code/platform. Try hitting the ping endpoint with both code and Postman/Insomnia to see if you're getting comparable timings or the code is much slower.  

If timings are similar, move onto calling the endpoints you're currently getting a slow response from and again compare calling using your code vs Postman or Insomnia.

**Reply by Esatyilmaz0** [reply]
I will change limit query params and speed is accelerated, When I tested it, the most appropriate limit value used to get the result of 100 receipts is 50 and 2 requests, so I was able to reduce the processing time from 10 seconds to 4 seconds.
